src/main/java/bio/overture/ego/config/DefaultProviderGuardListener.java

```diff
@@ -37,20 +36,17 @@ public void onApplicationEvent(ContextRefreshedEvent event) {
     checkState(isDefined(configuredProvider.toString()), "Default provider is not defined!");
     val storedProviderResult = defaultProviderService.findById(configuredProvider);
     // it is assumed that before boot the tripwire default provider has been set by running the
-    // flyway migration
+    // flyway migration. there will be a sql error if the table does not exist, indicating the
+    // migration has not run.
     checkState(
         storedProviderResult.isPresent(),
-        "Tripwire was not set! This means the flyway migration did not run yet.");
-    val storedProvider = storedProviderResult.get().getId();
-    checkState(
-        configuredProvider.equals(storedProvider),
-        "Configured default_provider '%s' does not match what was previously configured '%s'",
-        configuredProvider,
-        storedProvider);
+        "Configured default-provider '%s' does not match what was previously configured. Check the defaultprovidertripwire table.",
```

Comment:
it may be better, to output what was previously configured. this is more useful than "check the table". 
Configured default-provider '%s' does not match what was previously configured '%s)'`